### PR TITLE
 Addressed PA-498:Create multiple user and group

### PR DESCRIPTION
**Pull request by apimpalgaonkar**
What this PR does / why we need it:
Creates Multiple users and groups 

Which issue(s) this PR fixes (optional)
Closes #https://portworx.atlassian.net/browse/PA-498
Testrail - https://portworx.testrail.net/index.php?/cases/view/58032&group_by=cases:section_id&group_id=3897&group_order=asc&display_deleted_cases=0

### CODE REVIEW

**Review by vpinisetti-px — CHANGES_REQUESTED** [review]
Please add verification steps. Meaning verify users & groups are getting created or not by listing users & groups or any other mechanism such as logging into user etc..

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -312,6 +312,52 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
 
 }
 
+// This is to create multiple users and groups
+var _ = Describe("{CreateMultipleUserAndGroup}", func() {
+	numberOfUsers := 4
```
Comment (vpinisetti-px) [inline tests/backup/backup_test.go:317]: Better take large number of users & groups

File: tests/backup/backup_test.go [hunk 2]
```diff
@@ -312,6 +312,52 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
...
+	numberOfGroups := 2
+	users := make([]string, 0)
+	groups := make([]string, 0)
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:320]: please add starttorpedotest and endtorpedotest functions

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]

**Comment by mkoppal-px** [comment]
Please attach the aetos dashboard link after a successful run of this test

**Comment by apimpalgaonkar** [comment]
Created new PR

**Pull request closed without merging**